recipe/meta.yaml

```diff
@@ -44,11 +44,12 @@ requirements:
     - py-lief            # [not (win and py2k)]
     - pyyaml
     - scandir            # [py27]
+    - setuptools
     - six
     - glob2 >=0.6
     - pytz
     - tqdm
-    - conda-package-handling >=1.1
+    - conda-package-handling >=1.2
```

Comment:
```suggestion
    - conda-package-handling >=1.3
```

Reply:
Should we update [this line]( https://github.com/conda/conda-build/blob/14e89bee1ef6c0506bdee7d2aaae2fc48d231939/conda.recipe/meta.yaml#L48 ) as well?

Reply:
Added PR ( https://github.com/conda/conda-build/pull/3587 ).